website/content/en/docs/building-operators/helm/reference/advanced_features/reconcile-period.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Reconcile Period from Custom Resource Annotations
+linkTitle: Custom Resource Annotations Reconcile Period
+weight: 200
+description: Allow a user to set the desired reconcile period from the custom resource's annotations
+---
+
+While running a helm operator, we can specify the reconcile-period through the custom resource's annotations under the `helm.operator-sdk/reconcile-period` key. Using this feature will reconcile the cluster in a fixed interval of time. The reconcile period can be specified in the custom resource's annotations in the following manner: 
+
+```sh
+...
+metadata:
+  name: nginx-sample
+  annotations:
+    helm.operator-sdk/reconcile-period: "5s"
```

Comment:
What happens if the reconciliation period not be informed?
What is the default behaviour? 
Have we the same behaviour for Ansible and Helm in this case?

Reply:
If no reconcile period is specified in the CR annotations, then the default value is 1m as indicated [here](https://github.com/operator-framework/operator-sdk/blob/master/internal/helm/flags/flag.go#L60).

Yes, this behavior exists for ansible.

Reply:
Can be a follow up either, but should we not. ensure that same for Ansible?

Reply:
@camilamacedo86: So we are now going to accept reconcile Period through 3 ways:
1. from CR annotations 
3. command line flags 
3. through watches.yaml 
The order of precedence imo should be CR annotations > watches.yaml > cmd line flags. We want it to be in sync with ansible, but ansible does not accept it from flags and neither does the hybrid helm implementation. We would want to eventually remove accepting it through cmd line flags, but I think that should be a follow up. From discussion with @VenkatRamaraju, looks like CR annotations override values from `watches.yaml` in ansible. So I think we are good with that.